kore/src/Kore/Step/Simplification/TermLike.hs

```diff
@@ -277,6 +283,16 @@ simplifyInternal term predicate =
                 --
                 StringLiteralF stringLiteralF ->
                     return $ StringLiteral.simplify (getConst stringLiteralF)
+                InternalBytesF (Const
+                    Bytes.InternalBytes { bytesValue, string2BytesSymbol }) ->
+                    Application.simplify predicate =<< simplifyChildren
```

Comment:
This should get handed off to an internal simplifier instead.

Reply:
Does this mean just moving the code and calling `return $ Bytes.simplify`?

Reply:
Well, I don't think we need to do anything here. The value is already fully evaluated. We don't call the application simplifier for any other builtins. I think we should just return the value we were given.